docs/ABOUT.md

```diff
@@ -1,35 +1,7 @@
+[Java](https://go.java/index.html) is among the most popular available programming languages, thanks to its versatility and compatibility. It is widely used for software development, mobile applications, and large systems development.
 
-[Java](https://en.wikipedia.org/wiki/Java_(programming_language)) is a general-purpose computer programming language that is: 
- 1. [Concurrent](https://en.wikipedia.org/wiki/Concurrent_computing) 
- 2. [Class-Based](https://en.wikipedia.org/wiki/Class-based_programming)
- 3. [Object-Oriented](https://en.wikipedia.org/wiki/Object-oriented_programming)
+Besides it is not that young as the fresh languages out there, Java which was born in 1995 and is maintained by [Oracle](https://www.oracle.com/index.html) was designed to be fast, secure, reliable, beginner-friendly and highly portable. This portability perk exists as it must be execute through a cross-platform compatible [Java Virtual Machine - JVM](https://en.wikipedia.org/wiki/Java_virtual_machine). Android apps are also developer using Java, since the [Android Operating System](https://en.wikipedia.org/wiki/Android_(operating_system)) runs on a Java language environment.
```

Comment:
These sentences don't quite make sense in English. I would suggest something like:
"
Java was born in 1995 and is maintained by [Oracle](https://www.oracle.com/index.html). Despite the fact that it isn't as young as some of the fresh languages out there, Java is still really popular now. It was designed to be fast, secure, reliable, beginner-friendly and highly portable. This portability perk exists because Java is executed on a cross-platform compatible [Java Virtual Machine - JVM](https://en.wikipedia.org/wiki/Java_virtual_machine). Android apps are also developed using Java, since the [Android Operating System](https://en.wikipedia.org/wiki/Android_(operating_system)) runs on a Java language environment.
"

Reply:
I fixed the English mistakes as you suggested. 
Again, sorry for that. I'll take better care of this kind of situations :(